randovania/games/prime1/json_data/Phazon Mines.txt

```diff
@@ -1474,7 +1493,9 @@ Extra - aabb: [-108.57961, -160.31125, -168.22823, 77.58658, -50.59403, -92.4295
           Any of the following:
               Space Jump Boots or Slope Jump (Beginner)
```

Comment:
slope jump beginner was a part of the NSJ method video
https://youtu.be/MwycHUSwAaQ?t=9